Microservices/event-service/src/main/java/app/sportahub/eventservice/dto/request/LocationRequest.java

```diff
@@ -0,0 +1,40 @@
+package app.sportahub.eventservice.dto.request;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.mongodb.lang.Nullable;
+import jakarta.validation.constraints.NotBlank;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public record LocationRequest(@NotBlank(message = "Location name must be provided")
+                              String name,
+
+                              @NotBlank(message = "Street number must be provided")
```

Comment:
Can you make it nullable as well? Event can be taken place at a Park where no street is given.

Reply:
Good point, done